tls/s2n_kem.h

```diff
@@ -49,6 +49,9 @@ struct s2n_kem_params {
     struct s2n_blob public_key;
     struct s2n_blob private_key;
     struct s2n_blob shared_secret;
+    /* Store whether the client included the length prefix of the PQ and ECC Shares in their ClientHello, so that the
+     * server can match the client's behavior. For the client side, store whether it should send the length prefix. */
+    bool len_prefixed;
 };
```

Comment:
Do we ever set/check this value for the server params? I wonder if this is the right place for it, since we only need one copy for the connection, not one copy for every kem_params.

But it is probably just one extra byte.

Reply:
Ah wait, I think I see: it's set for both the client and the server kem params, but only on server connections. And the client and serer values should always match.

I'm still not sure we need two copies.